lib/data_update_scripts/20210622145212_backfill_forem_consumer_app_team_id.rb

```diff
@@ -0,0 +1,9 @@
+module DataUpdateScripts
+  class BackfillForemConsumerAppTeamId
+    def run
+      forem_app = ConsumerApps::FindOrCreateByQuery.call(app_bundle: ConsumerApp::FOREM_BUNDLE, platform: :ios)
+      forem_app.team_id = "R9SWHSQNV8"
+      forem_app.save
+    end
+  end
+end
```

Comment:
This script ensures existing Forem iOS Consumer Apps have the `team_id` value backfilled